### ISSUE TITLE
Warning: Unknown prop `styleName` on <div> tag.

### ISSUE THREAD

**Issue by emzero** [gajus/react-css-modules#155]
As of React 15.1.0 this warning popups in the console when using react-css-modules

```
Warning: Unknown prop styleName on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
```

Since we have to add the attribute `styleName` to make use of `react-css-modules` I don't know how to get rid of if. 
I'm not sure exactly when this error triggers, maybe you can remove the attribute before it renders?

**Comment by gajus** [comment]
Please search the existing issues and close this issue as a duplicate.

**Comment by emzero** [comment]
Fixed in v3.7.7

**Comment by Praveen-Invoscape** [comment]
How to fix it emzero
